baseline-class-uniqueness.lock

```diff
@@ -0,0 +1,550 @@
+# Danger! Multiple jars contain identically named classes. This may cause different behaviour depending on classpath ordering.
+# Run ./gradlew checkClassUniqueness --fix to update this file
+
+## runtimeClasspath
+[com.sun.istack:istack-commons-runtime, com.sun.xml.bind:jaxb-core]
```

Comment:
oof

Reply:
IS there a way we can avoid this? I think com.sun.* has come from adding intellij as a dependency

Reply:
that's true - this is probably a false positive from adding all of intellij